extensions/amp-animation/0.1/runners/native-web-animation-runner.js

```diff
@@ -100,6 +100,14 @@ export class NativeWebAnimationRunner extends AnimationRunner {
     });
   }
 
+  /**
+   * Returns whether it was initialized or not.
+   * @return {boolean}
+   */
+  isInitialized() {
```

Comment:
Let's make this private.

```suggestion
  /**
   * Returns whether it was initialized or not.
   * @return {boolean}
   * @private
   */
  isInitialized_() {
```